parser/src/main/scala/com/github/scalalab3/logs/parser/StubParser.scala

```diff
@@ -0,0 +1,17 @@
+package com.github.scalalab3.logs.parser
+
+import com.github.scalalab3.logs.common.query._
+
+import scala.util.{Failure, Success, Try}
+
+class StubParser extends QueryParser {
+
+  // feel free to add your mappings here
+  val possibleQueries = Map(
```

Comment:
Почему?

> 21 апр. 2016 г., в 12:33, mike <notifications@github.com> написал(а):
> 
> In parser/src/main/scala/com/github/scalalab3/logs/parser/StubParser.scala:
> 
> > @@ -0,0 +1,17 @@
> > +package com.github.scalalab3.logs.parser
> > +
> > +import com.github.scalalab3.logs.common.query._
> > +
> > +import scala.util.{Failure, Success, Try}
> > +
> > +class StubParser extends QueryParser {
> > +
> > +  // feel free to add your mappings here
> > +  val possibleQueries = Map(
> в других частях это не будет использоваться
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>